HousingRepairsOnlineApi/Gateways/IBlobStorageGateway.cs

```diff
@@ -1,9 +1,11 @@
-﻿using System.Threading.Tasks;
+﻿using System;
+using System.Threading.Tasks;
 
 namespace HousingRepairsOnlineApi.Gateways
 {
     public interface IBlobStorageGateway
     {
         Task<string> UploadBlob(string base64Img, string fileExtension);
+        Task<string> GetServiceSasUriForBlob(string fileName, int daysUntilExpiry, string storedPolicyName = null);
```

Comment:
The method name is implementation specific and binds implementations to use an Azure specific term.
Please name to be implementation agnostic, 
```suggestion
        Task<string> GetUriForBlob(string fileName, int daysUntilExpiry, string storedPolicyName = null);
```